dbbackup/management/commands/dbbackup.py

```diff
@@ -29,6 +29,8 @@ class Command(BaseDbBackupCommand):
         make_option("-s", "--servername", help="Specify server name to include in backup filename"),
         make_option("-z", "--compress", help="Compress the backup files", action="store_true", default=False),
         make_option("-e", "--encrypt", help="Encrypt the backup files", action="store_true", default=False),
+        make_option("-m", "--mongo", help="Backup a mongo db instead of sql. Uses MONGO_SETTINGS", action="store_true",
+                    default=False),
```

Comment:
It's just a suggestion to put `BACKUP_OPTIONS` directly in `DATABASES`, but it could as well be:

```python
BACKUP_DATABASES = {
    'my_mongo': {'temp_dir': '...'},
}
```